spec/systems/keyword_spec.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Keyword, type: :system do
+  describe 'redirects' do
+    context 'given an unauthenticated user' do
```

Comment:
no need to test this case on system test @gutakk :) this is the un-happy path, it's not a feature :) also the controller test handled that already, so we could remove it here... What do you think?

Reply:
Agree. I will make the change.

Reply:
Fixed in e44427aa5f5bbbdcce0eed76c8589c9c0fc4b9f9